Misc/NEWS.d/next/Library/2018-09-11-10-51-16.bpo-24412.i-F_E5.rst

```diff
@@ -0,0 +1,2 @@
+Add ``addModuleCleanup()`` and ``addClassCleanup()`` to unittest to support
```

Comment:
It may be worth to add references: ``` :meth:`~unittest.TestCase.addModuleCleanup()` ```, ``` :func:`~unittest.addModuleCleanup()` ```, ``` :meth:`~unittest.TestCase.setUpClass()` ```, ``` :func:`~unittest.setUpModule()` ```.